docs/getting_started/command_line.rst

```diff
@@ -0,0 +1,344 @@
+
+    They are put in your ``great_expectations/uncommitted`` directory and you can delete them at any time.
+
+    Because they can expose actual data, we stronly suggest leaving them in the ``uncommitted`` directory to avoid potential data leaks into source control.
+
+
+``great_expectations suite edit <SUITE_NAME> --no-jupyter``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you prefer to disable Great Expectations from automatically opening the generated jupyter notebook, add the ``--no-jupyter`` flag.
+
+.. code-block:: bash
+
+    $ great_expectations suite edit npi.warning --no-jupyter
+    To continue editing this suite, run jupyter notebook /Users/dickens/Desktop/great_expectations/uncommitted/npi.warning.ipynb
+
+You can then run juypter yourself.
+
+
+
+great_expectations datasource
+==============================
+
+All command line operations for working with datasources are here.
+A datasource is a connection to data and a processing engine.
+Examples of a datasource are:
+- csv files processed in pandas or Spark
+- a relational database such as Postgres, Redshift or BigQuery
+
+``great_expectations datasource list``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This command displays a list of your datasources and their types.
+These can be found in your ``great_expectations/great_expectations.yml`` config file.
+
+.. code-block:: bash
+
+    $ great_expectations datasource list
+    [{'name': 'files_datasource', 'class_name': 'PandasDatasource'}]
+
+
+``great_expectations datasource new``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This interactive command helps you connect to your data.
+
+.. code-block:: bash
+
+    $ great_expectations datasource list
+    What data would you like Great Expectations to connect to?
+    1. Files on a filesystem (for processing with Pandas or Spark)
+    2. Relational database (SQL)
+    : 1
+
+    What are you processing your files with?
+        1. Pandas
+        2. PySpark
+    : 1
+
+    Enter the path (relative or absolute) of the root directory where the data files are stored.
+    : data
+
+    Give your new data source a short name.
+     [data__dir]: npi_drops
+    A new datasource 'npi_drops' was added to your project.
+
+If you are using a database you will be guided through a series of prompts that collects and verifies connection details and credentials.
+
+
+``great_expectations datasource profile``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For details on profiling, see this :ref:`reference document<profiling_reference>`
+
+.. caution:: Profiling is a beta feature and is not guaranteed to be stable. YMMV
+
+
+Miscellaneous
+======================
+
+* ``great_expectations project check`` checks your ``great_expectations/great_expectations.yml`` for validity. This is handy for version migrations.
```

Comment:
"This is handy for version migration" - I know exactly what you mean, but I suspect a new external user would be confused. Maybe add a couple of sentences why and when I should consider running this command?